src/app/components/ng-class/ng-class.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-ng-class',
+  templateUrl: './ng-class.component.html',
+  styleUrl: './ng-class.component.scss'
+})
+export class NgClassComponent {
+
+  public active: boolean = true;
+
+  public changeColor(){
+    if(this.active)
```

Comment:
esta manera es correcta en cuanto a funcionalidad, pero una mejor manera sería en una sola línea así:
this.active = !this.active;